src/core/main.cpp

```diff
@@ -445,47 +429,52 @@ void print_xlio_global_settings()
     time_t clock = time(nullptr);
     char ofed_version_info[MAX_VERSION_STR_LEN];
 
-    vlog_printf(VLOG_INFO,
-                "---------------------------------------------------------------------------\n");
-    vlog_printf(VLOG_INFO, "%s\n", xlio_version_str);
+    __log_info("---------------------------------------------------------------------------\n");
+    __log_info("%s\n", xlio_version_str);
     if (PRJ_GIT_VERSION[0]) {
-        vlog_printf(VLOG_INFO, "%s\n", "Git: " PRJ_GIT_VERSION);
+        __log_info("%s\n", "Git: " PRJ_GIT_VERSION);
     }
-    vlog_printf(VLOG_INFO, "Cmd Line: %s\n", safe_mce_sys().app_name);
+    __log_info("Cmd Line: %s\n", safe_mce_sys().app_name);
 
     // Use DEBUG level logging with more details in RPM release builds
     vlog_levels_t log_level = VLOG_DEBUG;
 #if !defined(PRJ_LIBRARY_RELEASE) || (PRJ_LIBRARY_RELEASE == 0)
     // If non RPM (development builds) use more verbosity
     log_level = VLOG_DEFAULT;
 #endif
-    vlog_printf(log_level, "Current Time: %s", ctime(&clock));
-    vlog_printf(log_level, "Pid: %5u\n", getpid());
+    __log_raw(log_level, "Current Time: %s", ctime(&clock));
+    __log_raw(log_level, "Pid: %5u\n", getpid());
 
     ofed_version_info[0] = '\0';
     int ret = get_ofed_version_info(ofed_version_info, MAX_VERSION_STR_LEN);
     if (!ret && strlen(ofed_version_info) > 0) {
-        vlog_printf(VLOG_INFO, "OFED Version: %s\n", ofed_version_info);
+        __log_info("OFED Version: %s\n", ofed_version_info);
     }
 
     if (!uname(&sys_info)) {
-        vlog_printf(VLOG_DEBUG, "System: %s\n", sys_info.release);
-        vlog_printf(log_level, "Architecture: %s\n", sys_info.machine);
-        vlog_printf(log_level, "Node: %s\n", sys_info.nodename);
+        __log_dbg("System: %s\n", sys_info.release);
+        __log_raw(log_level, "Architecture: %s\n", sys_info.machine);
+        __log_raw(log_level, "Node: %s\n", sys_info.nodename);
     }
 
-    vlog_printf(VLOG_INFO,
-                "---------------------------------------------------------------------------\n");
+    __log_info("---------------------------------------------------------------------------\n");
 
     if (safe_mce_sys().mce_spec != MCE_SPEC_NONE) {
-        vlog_printf(VLOG_INFO, FORMAT_STRING, "Spec",
-                    xlio_spec::to_str((xlio_spec_t)safe_mce_sys().mce_spec), SYS_VAR_SPEC);
+        __log_info(FORMAT_STRING, "Spec", xlio_spec::to_str((xlio_spec_t)safe_mce_sys().mce_spec),
+                   SYS_VAR_SPEC);
+    }
+
+    if (log_level ::to_str(safe_mce_sys().log_level) &&
+        strcmp(log_level ::to_str(safe_mce_sys().log_level), "")) {
+        doca_log(DOCA_LOG_LEVEL_INFO, log_source,
+                 "/.autodirect/mtrswgwork/tcabouly/libxlio/src/core/main.cpp", 505, __func__,
```

Comment:
?

Reply:
sorry! tried to find all the debug things I did but missed this one